libs/common/src/assets/i18n/de.json

```diff
@@ -0,0 +1,892 @@
+{
+	"account.account_name_invalid": "Ungültiger Kontoname",
+	"account.accumulated_consumption": "Kumulierter Tagesverbrauch nach Funktion",
+	"account.active_config": "Aktive Konfiguration",
+	"account.add_stash": "Wissensbox hinzufügen",
+	"account.add_user": "Nutzer hinzufügen",
+	"account.admin.make_admin": "Administrator erstellen",
+	"account.admin.warning": "Wenn Sie diesem Benutzer die Administratorrolle zuweisen, kann er auf alle Optionen zugreifen und diese bearbeiten.",
+	"account.billing": "Zahlung & Abonnement",
+	"account.chart_error_media_seconds": "Die Daten zu „Medienstunden“ konnten nicht abgerufen werden",
+	"account.chart_error_processing_status": "Die Daten „zur Verarbeitung ausstehende Ressourcen“ konnten nicht abgerufen werden",
+	"account.chart_error_searches": "„Suchdaten“ können nicht abgerufen werden",
+	"account.chart_error_train_seconds": "Die Daten zu „Zugstunden“ können nicht abgerufen werden",
```

Comment:
This one translated "train seconds" to "train hours" but train as in the transportation method 🚂 . It should probably use the english word training like "Trainingsekunden".

Reply:
same here, english is `Unable to fetch "Train hours" data`. I guess a better english translation would be `Unable to fetch "Training hours" data`, so the translator then doesn't think we're talking about trains (we had the same problem with french)